pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java

```diff
@@ -245,7 +245,34 @@ public void testSingleColumnDistinctOnlyInnerSegment()
       // String columns
       //@formatter:off
       List<String> queries = Arrays
```

Comment:
Since there is only one query, let's use a single value instead of a list. We can also remove the comment to turn off/on the formatter